launcher/install_prereqs.cmake.in

```diff
@@ -23,5 +23,5 @@ set(gp_cmd_paths ${gp_cmd_paths}
 )
 
 include(BundleUtilities)
-fixup_bundle("@APPS@" "${QTPLUGINS}" "@DIRS@")
```

Comment:
This looks like it breaks packaging.

Reply:
Seems it wasn't needed to delete that line after all.